app/routers/user.py

```diff
@@ -59,3 +59,38 @@ def get_all_user_events(session: Session, user_id: int) -> List[Event]:
         session.query(Event).join(UserEvent)
         .filter(UserEvent.user_id == user_id).all()
     )
+
+
+def disable_user(session: Session, user_id: int) -> bool:
```

Comment:
These functions look like an internal functions. Please move them to `app/internal` and add `/user/enable` and `/user/disable`.

Also, please check that only the user itself/the administrator does the disabling/enabling.

Reply:
added this feature- but currently commented it since the logger system is not built yet, and I can't mock logged user without it :)